.travis.yml

```diff
@@ -1,13 +1,20 @@
+sudo: required
 dist: trusty
+addons:
+  chrome: stable
+  firefox: latest
 language: node_js
 node_js:
 - 10
 cache:
 - node_modules
+before_install:
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
 install:
 - npm install
 script:
-- npm t -- --browsers=BrowserStack_Chrome,BrowserStack_Firefox,BrowserStack_Safari,BrowserStack_Edge
+- npm t -- --browsers=Chrome,Firefox,BrowserStack_Safari,BrowserStack_Edge
```

Comment:
Hmm, shouldn't we remove `--browsers` parameter altogether and delegate getting browsers to `karma.conf.js`? 🤔 AFAIK passing the list of browsers directly into command will bypass the list prepared by config. Therefore this change will still not work on community PRs.

Reply:
Good point, I have missed this one 🙈